responsibleai/responsibleai/managers/explainer_manager.py

```diff
@@ -179,9 +179,54 @@ def get_data(self):
         :rtype: List[ModelExplanationData]
         """
         return [
-            self._get_interpret(i) for i in self.get()]
+            self._get_interpret(
+                i, self._evaluation_examples) for i in self.get()]
+
+    def request_explanations(self, local: bool, data: Any):
+        """Return the explanations for a given point(s) .
+
+        :param local: True if local explanations are quested
+                      and False otherwise.
+        :type local: bool
+        :param data: The data point(s) for which the explanations
+                     need to be generated.
+        :type data: Any
+        :return: The serialized version explanations for the
+                 given data point(s).
+        :rtype: Any
```

Comment:
I think there is some BaseExplanation type that can be used here to be more specific than Any

Reply:
Changed it to be ModelExplanationData.